### ISSUE TITLE
How to test equivalent of post form action behaviour

### ISSUE THREAD

**Issue by evergreenwebsystems** [calebporzio/livewire#91]
Apologies if this is super obvious. How would I test the following simple form like Livewire component which is used to store some data in the database from some input fields? Can I even test it this way, I am basically just trying to mimic how I would test a standard Laravel controller.

Livewire component (Working)
```
class Create extends Component
{
    public $title;
    public $description;
    public $role_id;

    public function createProject()
    {
        $attributes = $this->validate([
            'title' => 'required|min:10',
            'description' => 'required|min:10',
        ]);

        $project = Project::create($attributes);
        $project->users()->attach(auth()->user(), ['role_id' => $this->role_id]);

        $this->emit('renderProjects');
        $this->emit('closeModal');
    }

    public function render()
    {
        return view('livewire.projects.create');
    }
}
```

The following does not work. I can see my `$project` instance but I have no idea how to correctly do the equivalent of posting the new project data to the `createProject` method.

Test
```
    /** @test */
    public function a_user_can_create_a_project()
    {
        /** login auth user */
        $this->signIn();

        /** assert page exists */
        $this->get('/projects')->assertStatus(200);

        /** new project data */
        $data = [
            'title' => $this->faker->sentence,
            'description' => $this->faker->paragraph,
            'role_id' => 1,
        ];

        /**  livewire project create class */
        $project = Livewire::test(Create::class);

        /** PUSH DATA AND RUN METHOD??? */
        $project->createProject($data);

        /** assert project in database */
        $this->assertDatabaseHas('projects', [
            'title' => $data['title'],
            'description' => $data['description'],
        ]);

        /** assert project in page */
        $this->get('/projects')->assertSee($data['title']);
    }
```

**Comment by calebporzio** [comment]
What you are looking for is the "->set()" method.

I just added this to the "Testing" section of the docs: https://livewire-framework.com/docs/testing/

![image](https://user-images.githubusercontent.com/3670578/62241369-08cbe000-b3a7-11e9-9472-6940c9da1383.png)

Now, you can set all the data you need, and THEN call the "createProject" method, and it will have the proper data. Make sense?

**Comment by evergreenwebsystems** [comment]
Hi Caleb. That works great, however is it possible to pass in an array instead of having to set each input individually? Thanks

**Comment by calebporzio** [comment]
Not currently, but I just added support, so it will be available in the next release (v0.1.1)
